requirements.txt

```diff
@@ -7,3 +7,6 @@ python-dateutil==2.8.2
 lxml==5.1.0
 elementpath==4.4.0
 lorem-text==2.1
+jmespath==1.0.1
+rdflib == 7.0.0
+StrEnum == 0.4.15
```

Comment:
Should already be in Python 3.11 which we are using. So can also be removed I guess?